Makefile

```diff
@@ -171,11 +173,23 @@ test-with-cover: install-go-tools dashboard-ui
 	@$(FAILPOINT_ENABLE)
 	for PKG in $(TEST_PKGS); do\
 		set -euo pipefail;\
-		CGO_ENABLED=1 GO111MODULE=on go test -race -covermode=atomic -coverprofile=coverage.tmp -coverpkg=./... $$PKG  2>&1 | grep -v "no packages being tested" && tail -n +2 coverage.tmp >> covprofile || { $(FAILPOINT_DISABLE); rm coverage.tmp && exit 1;}; \
+		CGO_ENABLED=1 GO111MODULE=on go test -race -covermode=atomic -coverprofile=coverage.tmp -coverpkg=./... $$PKG 2>&1 | grep -v "no packages being tested" && tail -n +2 coverage.tmp >> covprofile || { $(FAILPOINT_DISABLE); rm coverage.tmp && exit 1;}; \
 		rm coverage.tmp;\
 	done
 	@$(FAILPOINT_DISABLE)
 
+test-with-cover-parallel: install-go-tools dashboard-ui split
+
+	@$(FAILPOINT_ENABLE)
+
+	set -euo pipefail;\
+
+	CGO_ENABLED=1 GO111MODULE=on gotestsum --junitfile test-report.xml -- -v --race -covermode=atomic -coverprofile=coverage $(shell cat package.list)  2>&1 || { $(FAILPOINT_DISABLE); }; \
```

Comment:
BTW, do we need to ignore these generated files?

Reply:
it do not need to remove because only dailyCI will run it and analyze them .

Reply:
I mean in our .gitignore

Reply:
yes，it is better to do it .